### PR TITLE
Smacks Spans into line, ohfuck

### PR DESCRIPTION
**Pull request by Shadow-Quill**
## About The Pull Request

Whammies a whole load of loose `<span class='...'>` into the appropriate defines.
Selection was done with regex, and fixing was done by hand. Some spans have been left alone, as they're fairly 'complex', and I didn't want to spend half an hour on each figuring out parentheticals.

### CODE REVIEW

**Comment by Shadow-Quill** [comment]
tfw I self-conflict

File: code/game/gamemodes/technomancer/instability.dm [hunk 1]
```diff
@@ -283,7 +283,7 @@
 	amount = amount * armor_factor
 	if(amount && prob(10))
 		if(isSynthetic())
-			to_chat(src, "<span class='cult'><font size='4'>Warning: Anomalous field detected.</font></span>")
+			to_chat(src, SPAN_CULT("<font size='4'>Warning: Anomalous field detected.</font>"))
```
Comment (Zandario) [inline code/game/gamemodes/technomancer/instability.dm:286]:
``` suggestion
			to_chat(src, SPAN_CULTLARGE("Warning: Anomalous field detected."))
```

File: code/game/gamemodes/technomancer/instability.dm [hunk 2]
```diff
@@ -283,7 +283,7 @@
...
 		else
-			to_chat(src, "<span class='cult'><font size='4'>The purple glow makes you feel strange...</font></span>")
+			to_chat(src, SPAN_CULT("<font size='4'>The purple glow makes you feel strange...</font>"))
```
Comment (Zandario) [inline code/game/gamemodes/technomancer/instability.dm:288]:
``` suggestion
			to_chat(src, SPAN_CULTLARGE("The purple glow makes you feel strange..."))
```

File: code/game/gamemodes/game_mode.dm [hunk 3]
```diff
@@ -466,7 +466,7 @@ var/global/list/additional_antag_types = list()
 //Reports player logouts//
 //////////////////////////
 /proc/display_roundstart_logout_report()
-	var/msg = "<span class='notice'><b>Roundstart logout report</b>\n\n"
+	var/msg = SPAN_NOTICE("<b>Roundstart logout report</b>\n\n")
```
Comment (Zandario) [inline code/game/gamemodes/game_mode.dm:469]:
``` suggestion
	var/msg = SPAN_BOLDNOTICE("Roundstart logout report\n\n")
```

File: code/game/machinery/camera/camera.dm [hunk 4]
```diff
@@ -121,7 +121,7 @@
 	if (istype(AM, /obj))
 		var/obj/O = AM
 		if (O.throw_force >= src.toughness)
-			visible_message("<span class='warning'><B>[src] was hit by [O].</B></span>")
+			visible_message(SPAN_WARNING("<B>[src] was hit by [O].</B>"))
```
Comment (Zandario) [inline code/game/machinery/camera/camera.dm:124]:
``` suggestion
			visible_message(SPAN_BOLDWARNING("[src] was hit by [O]."))
```

File: code/game/machinery/camera/camera.dm [hunk 5]
```diff
@@ -213,7 +213,7 @@
 		user.setClickCooldown(user.get_attack_speed(W))
 		if (W.force >= src.toughness)
 			user.do_attack_animation(src)
-			visible_message("<span class='warning'><b>[src] has been [W.get_attack_verb(src, user)] with [W] by [user]!</b></span>")
+			visible_message(SPAN_WARNING("<b>[src] has been [W.get_attack_verb(src, user)] with [W] by [user]!</b>"))
```
Comment (Zandario) [inline code/game/machinery/camera/camera.dm:216]:
``` suggestion
			visible_message(SPAN_BOLDWARNING("[src] has been [W.get_attack_verb(src, user)] with [W] by [user]!"))
```

File: code/game/machinery/computer/shuttle.dm [hunk 6]
```diff
@@ -40,22 +40,22 @@
 				if (src.auth_need - src.authorized.len > 0)
 					message_admins("[key_name_admin(user)] has authorized early shuttle launch")
 					log_game("[user.ckey] has authorized early shuttle launch")
-					to_chat(world, "<span class='notice'><b>Alert: [auth_need - authorized.len] authorizations needed until shuttle is launched early</b></span>")
+					to_chat(world, SPAN_NOTICE("<b>Alert: [auth_need - authorized.len] authorizations needed until shuttle is launched early</b>"))
```
Comment (Zandario) [inline code/game/machinery/computer/shuttle.dm:43]:
``` suggestion
					to_chat(world, SPAN_BOLDNOTICE("Alert: [auth_need - authorized.len] authorizations needed until shuttle is launched early"))
```

File: code/game/machinery/computer/shuttle.dm [hunk 7]
```diff
@@ -40,22 +40,22 @@
...
+					to_chat(world, SPAN_NOTICE("<b>Alert: Shuttle launch time shortened to 10 seconds!</b>"))
 					SSemergencyshuttle.set_launch_countdown(10)
 					//src.authorized = null
 					qdel(src.authorized)
 					src.authorized = list(  )
 
 			if("Repeal")
 				src.authorized -= W:registered_name
-				to_chat(world, text("<span class='notice'><b>Alert: [] authorizations needed until shuttle is launched early</b></span>", src.auth_need - src.authorized.len))
+				to_chat(world, text(SPAN_NOTICE(SPAN_BOLD("Alert: [src.auth_need - src.authorized.len] authorizations needed until shuttle is launched early"))))
```
Comment (Zandario) [inline code/game/machinery/computer/shuttle.dm:55]:
``` suggestion
				to_chat(world, SPAN_BOLDNOTICE("Alert: [src.auth_need - src.authorized.len] authorizations needed until shuttle is launched early"))
```

**Comment by Zandario** [comment]
FUCK, I hit it too early, but yeah I was going to review it all but fuck that, I want to do more than one thing today

**Comment by Shadow-Quill** [comment]
github, why the fuck can't I batch changes from the conversation page

**Pull request closed without merging**